.tipi/deps

```diff
@@ -0,0 +1,6 @@
+{
+	"requires": {
+		"tipi-deps/Catch2@file://single_include": { "@": "v2.x" },
+		"nlohmann/json": { "@": "v3.10.5" }
+	}
+}
```

Comment:
What are the semantics for tipi dependency resolution? toml++ only requires these for running tests; regular users have no need for these depedencies at all.

Reply:
That's a point we're currently working on. We are trying a bunch of variants to find something that works nicely for many cases (there's some headaches involved when you have repository "features" that collide with each other, and we want to gracefully handle some edge cases). For now the users have to pull the deps anyway but the perf hit is really there only for the first build, after that its cached. Not perfect (yet) but it works

Reply:
Ah yup. Sounds reasonable.

Reply:
> here's some headaches involved when you have repository "features" that collide with each other

C++ tooling in a nutshell

Reply:
Yeah. But we're pretty confident that we'll turn up with something actually nice. We try hard to not just shift aways the ugly things to our users but rather to find sane defaults that are nice to live with ;)